e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -88,7 +89,20 @@ context('When fee record correction feature flag is enabled', () => {
 
       cy.clickContinueButton();
 
-      cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}/check-the-information`));
+      //---------------------------------------------------------------
+      // "Check the information" page
+      //---------------------------------------------------------------
+      cy.assertText(mainHeading(), 'Check the information before submitting the record correction request');
```

Comment:
can this be in a different it statement? easier to follow if something fails

Reply:
I find that running cypress tests locally, the ui is really clear which step of the test failed and why so I don't think this is a concern.

Reply:
Can it still be split up please, long it blocks are hard to manage in the future as well.  Since the test is asserting values on a page, it should have its own block.  It won't take too much longer to run, can keep getting to the page in a beforeEach.

Reply:
We don't just have to 'get to the page', we have to wipe the database and then rerun the seeding and then navigate to the page again. Then if something early in the flow fails you will have loads of failing tests not just one and they will be failing in the before each rather than even in the test which is very far from ideal.

Reply:
What you are suggesting is actually an anti-pattern when it comes to cypress best practices: https://docs.cypress.io/app/core-concepts/best-practices#Creating-Tiny-Tests-With-A-Single-Assertion

Reply:
> long it blocks are hard to manage in the future as well

Lots of nested describe blocks with set up split across multiple beforeEach's is harder to maintain

Reply:
I have split this up into several smaller tests in what I think are logical splits of things that could fail. Let me know if this is better!